applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.c

```diff
@@ -136,6 +141,12 @@ int bt_rend_init(void)
 			LOG_WRN("Failed to initialize VCS renderer: %d", ret);
 			return ret;
 		}
+
+		cfgs_enabled = true;
+	}
+
+	if (!cfgs_enabled) {
+		LOG_WRN("Both VCS controller and VCS renderer are disabled");
```

Comment:
This will print warning prints every time you compile for BIS. The way these functions were designed was that if the services are not enabled it will just pass through them without issues. The reason being that if you use bt_rend_vol_up and no services are enabled you will publish directly to zbus

Reply:
Removed this print